ch_tools/chadmin/cli/table_metadata.py

```diff
@@ -0,0 +1,93 @@
+import re
+import uuid
+
+UUID_PATTERN = "UUID"
+OFFSET_UUID_PATTERN = 1
+ENGINE_PATTERN = "ENGINE"
+
+PARSER_SUPPORT_TABLE_ENGINE = [
+    "MergeTree",
+    "ReplacingMergeTree",
+    "SummingMergeTree",
+    "AggregatingMergeTree",
+    "CollapsingMergeTree",
+    "VersionedCollapsingMergeTree",
+    "ReplicatedMergeTree",
+]
+
+
+class TableMetadata:
```

Comment:
This class breaks `single responsibility`. Suggest refactoring:
```
class TableMetadata -> class which only keeps parsed metadata.
class MetadataParser -> class responsible for metadata parsing.
```

Reply:
Good thought. I think TableMetadataParser is better.